packages/site/cms-site/src/errorHandler/ErrorHandlerProvider.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
+
+import * as React from "react";
+
+export const ErrorHandlerContext = React.createContext<{ onError: (error: Error) => void }>({
+    onError: () => {
+        // empty default function
+    },
```

Comment:
Should we define some default behavior here? For instance, throw the error during development?

Reply:
done in: 9e93554a99ca759e53351d4356c0d6155c50ecc0